core/sodasql/cli/cli.py

```diff
@@ -525,6 +525,24 @@ def scan(scan_yml_file: str, warehouse_yml_file: str, variables: tuple, time: st
     default=None,
     type=Path,
 )
+@click.option(
```

Comment:
it would be nice to move these arguments to a sub command:

```
soda ingest dbt
```

And only show the dbt flag when the `dbt` is given.

Reply:
I will have a go at this in a separate PR. With argparse I would do this by adding a [sub parser](https://docs.python.org/3/library/argparse.html#argparse.ArgumentParser.add_subparsers). Most likely `click` has a similar option

Reply:
Sounds good to me yeah, seems to be more of an improvement on design rather than a functional need for this PR. I don't really know click well (nor am I a fan of it so I've stayed away from it)

Reply:
See [here](https://github.com/sodadata/soda-sql/pull/608)